### PR TITLE
Make some projects for 'hour of code'

### PR DESCRIPTION
**Pull request by ganicke**
Let me know if some extra changes are needed to support the tutorial format.

### CODE REVIEW

File: docs/tutorials/hour-of-code/hey-microbit.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+})
+```
+
+## Step 10
+
+Now, click the **SHAKE** spot a few times to see how the @boardname@ is feeling. Hmm, it's mood is different sometimes!
+
+## Step 11
+
+Awesome job! You've completed your HoC: Microsoft MakeCode activity. If you have a @boardname@ to use, click the `|Download|` button and try your code on the board.
```
Comment (pelikhan) [inline docs/tutorials/hour-of-code/hey-microbit.md]: HoC -> Hour Of Code

**Comment by pelikhan** [comment]
we already have tutorials, why special case them for hour of code?

File: docs/tutorials/hour-of-code/hey-microbit.md (hunk 1 above)
Comment (pelikhan) [reply docs/tutorials/hour-of-code/hey-microbit.md]: This page should not mention it since we don’t have the branding footer rendered in docs

**Comment by microsoftsam** [comment]
I want these tutorials to be more "Simulator" specific, and have less of a focus on download like the other tutorials. Also microbit only has the one "getting started" tutorial. These tutorials will only be linked from the hourofcode page so we can churn them quicker.

**Comment by ganicke** [comment]
Want to explicitly mention "shake in the simulator" in the R-P-S project?

**Comment by microsoftsam** [comment]
Yeah that would be a good idea.

**Pull request merged**